dub.selections.json

```diff
@@ -0,0 +1,6 @@
+{
+	"fileVersion": 1,
+	"versions": {
+		"libdparse": "0.20.0"
```

Comment:
Why not in dub.sdl?
I guess, version 0.20.0 is required (otherwise, source code is silently skipped because of "errors"?) and not just recommended.

Reply:
Ja, daher hat dub.sdl auch ne Versionsanforderung an libdparse. Aber bei Binaries sollten die funktionierenden Versionen von Abhängigkeiten trotzdem festgeklopft sein.